### ISSUE TITLE
Quotes not allowed in CQL query

### ISSUE THREAD

**Issue by b2mdevelopment** [simplereach/helenus#69]
The following CQL works in cqlsh -3

```
CREATE KEYSPACE test WITH strategy_class='NetworkTopologyStrategy' AND strategy_options:"us-east"=2 
```

but not via Helenus, which raises the error

{ [HelenusInvalidRequestException: line 1:105 no viable alternative at character '"'] name: 'HelenusInvalidRequestException' }

Quotes have been allowed since [CASSANDRA-4278](https://issues.apache.org/jira/browse/CASSANDRA-4278?page=com.atlassian.jira.plugin.system.issuetabpanels:all-tabpanel).

**Comment by devdazed** [comment]
can you paste an example of how you are using Helenus to call this query?

**Comment by b2mdevelopment** [comment]
Sorry for not getting back to you earlier. We had difficulty reproducing exactly what we'd observed.

The create keyspace with quoted dash works fine in cql version 3 through helenus. 

We were seeing an intermittent problem when we changed cql version using the options json in helenus connect, and had difficulty reproducing it reliably. 

We have a need to switch between cql versions 2 and 3 due to a need to create dynamic counter columns (unavailable in 3) and creating keyspaces with datacenter names containing a dash (unavailable in 2, but required for the Ec2Snitch).

What we have discovered is that if we initially connect using cqlVersion 3 and then close and reconnect using cqlVersion 2 (without explicitly setting cqlVersion) the connection remains connected with cqlVersion: 3.0.0. 

We did notice, however, that if we explicitly specify the cql version every time (which is clearer anyway) this behaviour disappears.

It feels like an edge case, so we're not going to raise an issue. If you'd like any additional details let us know.

Regards,
[Roy](http://roylines.co.uk) and [James](https://github.com/jamesbloomer).
